blob/blob.go

```diff
@@ -792,6 +793,36 @@ func (b *Bucket) NewWriter(ctx context.Context, key string, opts *WriterOptions)
 	return w, nil
 }
 
+// Copy the blob stored at srcKey to dstKey.
+// A nil CopyOptions is treated the same as the zero value.
+//
+// If the source blob does not exist, Copy returns an error for which
+// gcerrors.Code will return gcerrors.NotFound.
+//
+// If the destination blob already exists, it is overwritten.
+func (b *Bucket) Copy(ctx context.Context, srcKey, dstKey string, opts *CopyOptions) (err error) {
```

Comment:
I think you should reverse src and dest, to be compatible with Go (e.g. the `copy` built-in).

Reply:
Good point, done.